src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="container">
+      <div class="calendar calendar--start-day--sun calendar--month-length--29">
+        <div class="calendar__cells"></div>
```

Comment:
The class for the individual cells should be 'calendar__cell' instead of 'calendar__cells' to follow BEM naming conventions.